core/src/main/java/com/nisovin/magicspells/MagicCommand.java

```diff
@@ -0,0 +1,720 @@
+package com.nisovin.magicspells;
```

Comment:
why was everything thrown into this file? That is the opposite of the direction we want to go

Reply:
I didn't get that impression. I was following up how other large systems are handled. How do you propose I extract this?